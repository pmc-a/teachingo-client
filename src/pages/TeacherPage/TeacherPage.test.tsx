import React from 'react';
import { shallow } from 'enzyme';

import Component from './TeacherPage';

jest.mock('../../state', () => ({
    useAppState: jest.fn().mockReturnValue({
        fetchLessons: jest.fn(),
    }),
}));

describe('TeacherPage', () => {
    it('should successfully render', () => {
        expect(shallow(<Component />)).toMatchSnapshot();
    });
});
