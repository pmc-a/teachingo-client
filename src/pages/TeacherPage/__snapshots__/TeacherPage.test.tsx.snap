// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeacherPage should successfully render 1`] = `
<div
  className="teacher-page-container"
>
  <h1>
    Lessons
  </h1>
  <div
    className="teacher-page-main-container"
  >
    <div
      className="lesson-stream-view"
    >
      <span>
        Lesson stream view
      </span>
      <div />
    </div>
    <LessonList
      lessons={Array []}
      setSelectedLesson={[Function]}
    />
  </div>
</div>
`;
